cdap-docs/admin-manual/source/installation/ambari/installing-cdap.rst

```diff
@@ -88,21 +88,12 @@ Installing CDAP
  
       **Ambari Dashboard:** Customizing Services 2
 
-#. If you wish to use the CDAP Explore service (to use SQL to query CDAP data), you must: have
-   Hive installed on the cluster; have the Hive client on the same host as CDAP; and set the
-   ``explore.enabled`` option to true.
-
-   .. figure:: ../../_images/ambari/ss07-config-enable-explore.png
-      :figwidth: 100%
-      :width: 800px
-      :align: center
-      :class: bordered-image
- 
-      **Ambari Dashboard:** Enabling *CDAP Explore*
-
    For a **complete explanation of these options,** refer to the :ref:`CDAP documentation of
-   cdap-site.xml <appendix-cdap-site.xml>`. After making any configuration changes, click
-   *Next*.
+      cdap-site.xml <appendix-cdap-site.xml>`. After making any configuration changes, click
+      *Next*.
```

Comment:
This will break; the formatting is wrong. There should be no indent on lines 92 & 93. But see my previous comment anyways.